pkg/action/install.go

```diff
@@ -204,9 +204,10 @@ func (i *Install) installCRDs(crds []chart.CRD) error {
 // Run executes the installation
 //
 // If DryRun is set to true, this will prepare the release, but not install it
-
 func (i *Install) Run(chrt *chart.Chart, vals map[string]interface{}) (*release.Release, error) {
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.TODO(), i.Timeout)
```

Comment:
Should this still be a context.Background()? It felt more appropriate to create a context with timeout, from the timeout field. I believe the idea is to replace Timeout completely, so maybe I should revert this?

Reply:
Updated to context.TODO() for Helm 4 cleanup.